tools/bazel.rc

```diff
@@ -61,17 +58,17 @@ build:tsan --copt=-fsanitize=thread
 build:tsan --copt=-fno-omit-frame-pointer
 build:tsan --copt=-DGPR_NO_DIRECT_SYSCALLS
 build:tsan --copt=-DGRPC_TSAN
-# TODO(jtattermusch): ideally we would set --copt=-DTHREAD_SANITIZER (used by absl)
```

Comment:
why isn't this TODO applicable anymore?

Reply:
Abseil doesn't use `DTHREAD_SANITIZER` anymore ([link](https://github.com/abseil/abseil-cpp/blob/d1de75bf540f091b4dfc860713d556e578c0f158/absl/base/config.h#L690)). Instead, it starts to detect it using a feature.